### PR TITLE
dsg

### PR DESCRIPTION
**Pull request by den99f**
@den99f

### CODE REVIEW

**Comment by den99f** [comment]
@me daf

File: static/sass/_wisdorm.scss [hunk 1]
```diff
@@ -0,0 +1,208 @@
+/****************************************************************************************************
+*   Stanford Theming File
+*
+*   Contains all the theme color definitions for the Stanford Open Source MOOC Project
```
Comment (den99f) [inline static/sass/_wisdorm.scss]: sadgdsagdsa

Comment (den99f) [inline static/sass/_wisdorm.scss]: > sadgdsagdsa

sdagsda

**Comment by stvstnfrd** [comment]
Hi @den99f . It looks like this PR was a mistake?

I'm going to close this for now; let me know if that's incorrect.

**Pull request closed without merging**